src/utility/Translation.ts

```diff
@@ -30,10 +30,16 @@ export class Translation {
   static CONNECTION_API_LIST_DESCRIPTION = `Add some calls to the exists Connection`;
 
   static NPM_INSTALL = `Start npm install this take a while`;
+
+  static FOLDER_IS_NOT_A_RELUTION_PROJECT(folder: string): string {
+    return `The folder "${folder}" is not a Relution project. Please read the manual "How to migrate exists Project into a relution Project".`;
+  }
+
   static SELECT(name: string): string {
     return `Select ${name} :`;
   }
-  static NOT_EMPTY(name:string):string {
+
+  static NOT_EMPTY(name: string): string {
     return `${name} can not be empty`;
```

Comment:
... must not be empty.

(Can ist im Englischen zwar "können", aber nicht im Sinne von darf-nicht. Darf-nicht ist i.d.R. ein must-not.)